packages/plugins/response-cache/src/plugin.ts

```diff
@@ -509,3 +548,51 @@ function calculateTtl(typeTtl: number, currentTtl: number | undefined): number {
 export const cacheControlDirective = /* GraphQL */ `
   directive @cacheControl(maxAge: Int) on FIELD_DEFINITION | OBJECT
 `;
+
+function mergePatch(
```

Comment:
Do you think we can use this function instead?
https://github.com/ardatan/graphql-tools/blob/master/packages/executors/http/src/handleMultipartMixedResponse.ts#L52